samples/Tests/Stride.Samples.Tests.csproj

```diff
@@ -1,6 +1,5 @@
 <Project>
   <!-- NuGet resolved needed for StrideDefaultAssetsPlugin.LoadDefaultTemplates() -->
-  <Import Project="..\..\sources\shared\Stride.NuGetResolver\Stride.NuGetResolver.projitems" Label="Shared" />
```

Comment:
Can you check if this was added by me as a workaround for the tests not running on a clean system?

Reply:
@manio143 Indeed, this was added to "auto resolve packages correctly" so I've undone that change.